### ISSUE TITLE
Sent Notifcations: A server error occured

### ISSUE THREAD

**Issue by bryantAXS** [verbb/formie#399]
**Description**
When navigation to the Sent Notifications screen, one specific form keeps showing an error when trying to load.

The response from the server is this:

`error: "An exception has been thrown during the rendering of a template ("Getting unknown property: verbb\formie\elements\SentNotification::preview")."`

https://share.getcloudapp.com/geu4927B

The form itself has a mixture of fields:
* Single Line Text
* Muli-line Text
* File Upload
* Dropbown
* Checkbox
* Entries

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.19.1
- Craft version: 3.15.1
- Multi-site: No

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.22",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Should be fixed in [1.3.23](https://github.com/verbb/formie/releases/tag/1.3.23)
